channel/http_callback_channel.go

```diff
@@ -0,0 +1,144 @@
+package channel
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/breez/notify/notification"
+	"github.com/gin-gonic/gin"
+	"github.com/google/martian/v3/log"
+)
+
+const (
+	callbackTimeout = 30 * time.Second
```

Comment:
I suggest to have a much greater timeout (at least 1 minute) as mobile devices may require some time to respond.

Reply:
Done [2bc0e63](https://github.com/breez/notify/pull/15/commits/2bc0e63573831ddabb18678011ccc5102e8907f7)

I would note the breez-lnurl service timeout is also 30 seconds